### ISSUE TITLE
Android trash icon is an emoji instead

### ISSUE THREAD

**Issue by SidonieBouthors** [laurent22/joplin#11202]
### Operating system

Android

### Joplin version

3.0.8

### Current behaviour

1. Install Joplin
2. Open sidebar
3. All icons are flat icons in the same style, but the trash icon is an emoji instead

![image](https://github.com/user-attachments/assets/d723fe45-00af-481a-a243-bb27566161de)

### Expected behaviour

I expected all icons to be in a similar or coherent style. This is the only one that is an emoji as far as I can tell.

**Comment by SidonieBouthors** [comment]
On a similar note, I also just installed Joplin on Pop!_OS and although the icon is not an emoji, the trash icon also looks different from all the others (it is a filled icon in a different style, instead of outline icons), but not sure if that is related.

**Desktop version info :** 

Joplin 3.0.15 (prod, linux)

Client ID: 9637ac3d343843ad9f3d9d74baf57297
Sync Version: 3
Profile Version: 47
Keychain Supported: No

**Current Behavior :** 

![image](https://github.com/user-attachments/assets/120a65f7-c0a1-4456-abef-4a73e5f555ca)

**Comment by mrjo118** [comment]
The icons for trash are clearly by design, but I guess you are questioning the choice of icons. I see your point on the desktop app about it not being entirely consistent, but to be honest I didn't notice until you mentioned it.

I would agree on the android app though that the emoji did seem out of place when the trash feature first appeared.

**Comment by SidonieBouthors** [comment]
Fair enough. The icon difference on desktop isn't too noticeable I just added it in case it was somehow related :) 
On android though I noticed it immediately (it also doesn't look good in dark mode since it doesn't change color like the other icons). I suppose if it wasn't changed earlier, it may have been a design choice to have it this way, but I can't quite understand what would be the use of bringing attention to the trash icon in particular.
